compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -574,6 +574,9 @@ void addFunctionType(DiagnosticPos pos, Set<Whitespace> ws, boolean paramsAvail,
         if (paramsAvail) {
             functionTypeNode.addWS(commaWsStack.pop());
             functionTypeNode.params.addAll(this.varListStack.pop());
+        } else if (restParamAvail) {
+            // VarListStack pops when function typed param has only rest params, Fix for: 20919
```

Comment:
We normally don't mention the issue number on fixes. Shall we remove the issue number?

Reply:
+1

Reply:
Okay. Thanks!